src/NLog/Targets/MailTarget.cs

```diff
@@ -284,13 +307,26 @@ protected override void Write(AsyncLogEventInfo[] logEvents)
         /// </summary>
         protected override void InitializeTarget()
         {
-
+            InitializeFrom();
```

Comment:
I had put that in the constructor because it is easier to Unit test this way. I'll work on rewriting the Unit Tests if you think it will be better to put it in `ConfigureMailClient` :)